pkg/api/dashboard.go

```diff
@@ -82,7 +82,7 @@ func dashboardGuardianResponse(err error) response.Response {
 // 404: notFoundError
 // 500: internalServerError
 func (hs *HTTPServer) GetDashboard(c *contextmodel.ReqContext) response.Response {
-	ctx, span := hs.tracer.Start(c.Req.Context(), "httpserver.GetDashboard")
+	ctx, span := hs.tracer.Start(c.Req.Context(), "api.GetDashboard")
```

Comment:
🤔 I'm not sure this rename is necessary - this is the `api` package, but the `HTTPServer` type... the naming idiom I've usually used is to use `<type>.<method>`... though I could go either way...

Reply:
After discussion we decided to keep <package>.<method> because of the way we use interfaces across the codebase